src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,265 @@
+package duke;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+/**
+ * Handles the list of tasks and related methods.
+ */
+public class TaskList {
+
+    /**
+     * The ArrayList that stores the task objects.
+     */
+    private ArrayList<Task> tasks;
+
+    /**
+     * Creates the task list as an ArrayList of tasks.
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<Task>();
+    }
+
+    /**
+     * Gets the ArrayList itself
+     *
+     * @return the ArrayList of tasks that the tasks are stored in
+     */
+    public ArrayList<Task> getTaskList() {
+        return tasks;
+    }
+
+    /**
+     * Returns the task with the specified index from the list
+     *
+     * @param index the index of the list to be retrieved
+     * @return the task with that index
+     */
+    public Task get(int index) {
+        return tasks.get(index);
+    }
```

Comment:
Perpahs the name of the variable could be more specific like "getTask" so it is easily understandable